draft-ietf-mls-protocol.md

```diff
@@ -2582,14 +2571,6 @@ MLSPlaintext.confirmation_tag =
     MAC(confirmation_key, GroupContext.confirmed_transcript_hash)
 ~~~~~
 
-<!-- OPEN ISSUE: It is not possible for the recipient of a handshake
-message to verify that ratchet tree information in the message is
-accurate, because each node can only compute the secret and private
-key for nodes in its direct path.  This creates the possibility
-that a malicious participant could cause a denial of service by sending a
-handshake message with invalid values for public keys in the ratchet
-tree. -->
-
```

Comment:
I'm inclined to have this documented somewhere, perhaps not as an open issue. Otherwise someone will claim they discovered an "attack".